src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,28 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    this.hidden = false;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  hide() {
+    this.hidden = true;
+  }
```

Comment:
How can this animal stop hiding? 🙂 Better to change `hidden` value to opposite here

Reply:
I was wrong, ignore this comment 🙃